sunpy/timeseries/metadata.py

```diff
@@ -221,13 +221,13 @@ def find_indices(self, time=None, colname=None, **kwargs):
 
     def find(self, time=None, colname=None, **kwargs):
         """
-        Find all metadata matching the given filters for datetime and/or column name.
+        Find all metadata matching the given filters for `~astropy.time.Time` and/or column name.
         Will return all metadata entries if no filters are given.
 
         Parameters
         ----------
-        time : `str` or `~datetime.datetime` optional
-            The string (parsed using the `~sunpy.time.parse_time`) or datetime
+        time : `str` or `astropy.time.Time` optional
```

Comment:
Do we want to `~` all `astropy.time.Time` as a rule?

Reply:
I thought seeing just `Time` would be confusing for new users.

Reply:
I think it's context dependant. I am in favour in full pathing it in this bit of the docstring.